### ISSUE TITLE
TFBartForSequenceClassification

### ISSUE THREAD

**Issue by uglyboxer** [huggingface/transformers#19653]
The Tensorflow version of `BartForSequenceClassification` seems to be missing.  I've been putting a port together for another use case.  Any interest in adding it to the repo?

**Comment by nayanukande** [comment]
can you explain it in detail.

**Comment by uglyboxer** [comment]
I have been working on a project for zero-shot classification using `facebook/bart-large-mnli`.  After an initial prototype using the transformers pipeline I began to work with the underlying Bart model directly.  As our project is in Tensorflow, I soon noticed there was no equivalent of `class BartForSequenceClassification` on the TF side.  

I have stitched it together: adding the classification head and loading the weights from the torch.bin file.   If you think it would be useful, I could put up a PR add it to https://github.com/huggingface/transformers/blob/main/src/transformers/models/bart/modeling_tf_bart.py  along side of the other TF ports for Bart.
